Makefile

```diff
@@ -3,20 +3,20 @@ test:
 testcov:
 	py.test --cov=ideascube/ --cov-report=term-missing
 develop:
-	pip install -r requirements-dev.txt
```

Comment:
AFAIK, debian python is python 2. Without virtualenv, you have to explicitly call python3 if you want to use it.

I'm not sure to understand your question : 

> Was your venv made with python3 and sourced?